app/javascript/search_form.js

```diff
@@ -97,10 +97,17 @@ function enableGeodistance() {
   geodistance_label.innerText = 'Close distance search';
 };
 
+// Note that this skips hidden inputs, which are used to toggle advanced and geospatial searches.
+function clearInputs(event) {
```

Comment:
Can we not use https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/reset because we don't want those hidden inputs cleared?

Reply:
`No,` `reset()` meets a different use case. It resets the form to its original values (i.e., what they were when the current page loaded). The requested feature is to clear all form values.